service/controller/resource/app/desired.go

```diff
@@ -161,39 +191,28 @@ func (r *Resource) newApp(appOperatorVersion string, cr apiv1alpha2.Cluster, app
 				annotation.ForceHelmUpgrade: strconv.FormatBool(appSpec.UseUpgradeForce),
 			},
 			Labels: map[string]string{
-				pkglabel.App:             appSpec.App,
+				label.AppKubernetesName:  appSpec.App,
```

Comment:
Note: the label changes from `app` to `app.kubernetes.io/name`. app is deprecated by upstream and we default the new label in app-admission-controller.